src/topojson/topojson.ml

```diff
@@ -242,8 +235,33 @@ module Make (J : Intf.Json) = struct
              ("arcs", J.array (J.array (J.array J.int)) arcs);
            ]
           @ properties_or_null properties
-          @ bbox_to_json_or_empty bbox
-          @ foreign_members)
+          @ bbox_to_json_or_empty bbox @ foreign_members)
+    end
+
+    module Accessor = struct
+      let get_point = function
+        | Topojson.Geometry.Point p -> Ok p
+        | _ -> Error (`Msg "Expected point")
+
+      let get_point_exn = function
+        | Topojson.Geometry.Point p -> p
+        | _ -> invalid_arg "Expected point"
+
+      let get_point = function
+        | Topojson.Geometry.Point p -> Ok p
+        | _ -> Error (`Msg "Expected point")
+
+      let get_point_exn = function
+        | Topojson.Geometry.Point p -> p
+        | _ -> invalid_arg "Expected point"
+
+      let get_point = function
```

Comment:
You need to change these from points to the other possible geometries such as multipolygon.

Reply:
Okay.

But this part 
```ocaml 
Topojson.Geometry.Point
```  
also gives error `Unbound module Topojson`

Reply:
Because you are in the topojson file, you are in the `Topojson` module so you need to remove that bit.